examples/simplicial_2d_dg/elixir_euler_sbp_triangular_mesh_flux_diff.jl

```diff
@@ -0,0 +1,53 @@
+# !!! warning "Experimental features"
+
+using StartUpDG
+using Trixi, OrdinaryDiffEq
+
+polydeg = 3
+rd = RefElemData(Tri(), SBP(), polydeg)
+
+volume_flux = flux_ranocha
+dg = DG(rd, nothing #= mortar =#,
+        SurfaceIntegralWeakForm(FluxLaxFriedrichs()), VolumeIntegralFluxDifferencing(volume_flux))
```

Comment:
We have a keyword constructor for the weak form surface integral that you can use like
```julia
SurfaceIntegralWeakForm(surface_flux=flux_ranocha)
```
If you do that, it will allow people to change the surface flux easily using `trixi_include`.

Reply:
Nice trick. I'll add a keyword constructor in the next round of commits.

Reply:
Odd, I get an error when I try to run `SurfaceIntegralWeakForm(surface_flux=flux_ranocha)`. I tried with both `dev` and `main` - any ideas?

Reply:
Looks like I messed up something - my fault, sorry. But maybe it would be a nice addition. What do you think, @sloede?

Reply:
You mean, add a kwargs ctor with just one required argument, i.e., something like
```julia
SurfaceIntegralWeakForm(; surface_flux) = SurfaceIntegralWeakForm(surface_flux)
end
```
? Yes, that would make a lot of sense!

Reply:
It's an easy enough change to squeeze into this PR. Should we also add similar ctors for `VolumeIntegralFluxDifferencing` or `SurfaceIntegralStrongForm`?

Reply:
I think we should just open an issue and not do everything in this PR (which is already quite big). I created #728 to track this (resolving this conversation here).